### ISSUE TITLE
About Multi Class Pose Estimation

### ISSUE THREAD

**Issue by yu9s** [ultralytics/ultralytics#4715]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

Thank you for your work.  May I ask if posture estimation supports multiple categories？

For example,  detecting vehicles and key points of license plates, I changed the back four points of the vehicle to 0。But the four key points of the license plate are not very effective, Can you help me? 

_No response_

**Comment by glenn-jocher** [comment]
@yu9s hello, thank you for your question and for articulating your problem so clearly. 

In terms of multi-class pose estimation, it's absolutely possible for YOLOv8 to support it. It's indeed a common use case to detect multiple keypoints of different categories, such as vehicles and their license plates.

From your description, it seems you're experiencing an issue with detecting the four points of the license plate. There could be several reasons why this is happening. Your keypoints detection could be influenced by factors such as data quality, model optimization, or training settings.

When you train the model, ensure that you use a diverse dataset that includes enough representative samples of the key points you want to detect. Specifically ensure you have a sufficient number of quality examples of license plate keypoints. Additionally, you may also need to adjust the model parameters to improve the keypoints detection performance.

We hope this helps you in moving forward with your project and please don't hesitate to reach out on this platform should you have any additional queries or concerns. Your enthusiasm and creativity in exploring the possibilities of our YOLOv8 model are very much appreciated.

**Comment by yu9s** [comment]
> @yu9s hello, thank you for your question and for articulating your problem so clearly.
> 
> In terms of multi-class pose estimation, it's absolutely possible for YOLOv8 to support it. It's indeed a common use case to detect multiple keypoints of different categories, such as vehicles and their license plates.
> 
> From your description, it seems you're experiencing an issue with detecting the four points of the license plate. There could be several reasons why this is happening. Your keypoints detection could be influenced by factors such as data quality, model optimization, or training settings.
> 
> When you train the model, ensure that you use a diverse dataset that includes enough representative samples of the key points you want to detect. Specifically ensure you have a sufficient number of quality examples of license plate keypoints. Additionally, you may also need to adjust the model parameters to improve the keypoints detection performance.
> 
> We hope this helps you in moving forward with your project and please don't hesitate to reach out on this platform should you have any additional queries or concerns. Your enthusiasm and creativity in exploring the possibilities of our YOLOv8 model are very much appreciated.

Thank you for your answer.  As you said，I want my model to output the vehicle box and license plate keypoints, 

Now, I try to train it 600 epoches, reduce lr and reduce pose loss gain.

**Comment by glenn-jocher** [comment]
@yu9s, that sounds like a logical next step in your project. Training for more epochs can help to improve the performance of the model, and adjusting learning rate could potentially result in better convergence. 

Modifying the pose-loss gain is another valid approach to try. You can tweak this parameter until you see an improvement in the detection of key points on the license plate.

Please monitor your model's performance closely while making these adjustments. Remember that it can be a process of trial and error to obtain the optimal parameter settings for a specific task like this one.

Remember to ensure your data set is sufficiently diverse and representative of all the key points you want to detect to achieve the best performance. Data quality can be crucial in a task like this.

I hope these suggestions assist you in improving your model's performance, and I wish you the best of success with your project. Your ongoing patience and dedication to optimizing the YOLOv8 model for your specific use case are appreciated. Please, don't hesitate to reach out if you have further questions.
